net-libs/mbedtls/mbedtls-2.28.9-r1.ebuild

```diff
@@ -17,6 +17,7 @@ IUSE="cmac cpu_flags_x86_sse2 doc havege programs static-libs test threads zlib"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
+	programs? ( !!net-libs/mbedtls:0[programs] )
```

Comment:
!! is really hostile to users if we can help it. Do we really need double !, not just one?